single-node-k8s.sh

```diff
@@ -61,6 +61,7 @@ function set_env() {
     # it and will cause a failure if not annotated with /dev/null.
     # shellcheck source=/dev/null
     source "${GOPATH}"/src/github.com/ceph/ceph-csi/build.env
+    CSI_IMAGE_VERSION=${CSI_IMAGE_VERSION:-"canary"}
```

Comment:
```suggestion
    CSI_IMAGE_VERSION="${CSI_IMAGE_VERSION:-"canary"}"
```

Reply:
@nixpanic PTAL

Reply:
I do not think this is needed, `shellcheck` does not complain about it